### PR TITLE
[ADD][GRAP] custom mutual insurance

### PR DESCRIPTION
**Pull request by quentinDupont**
**if you think you're interested in these changes, I can integrate it into the existing modules :)**

In our cases, the person could be independent for his mutual, so three choices :
- At start : Don't know yet
- Then : Affiliated
- or : Exempted (and we should check if we have a proven file attached)

WIth exempted choices, we don't need the other fields

affialiated
or
exempted

### CODE REVIEW

**Comment by robinkeunen** [comment]
@djulien19 fyi

**Review by robinkeunen — APPROVED** [review]
LGTM 👌 

I tagged Julien to see if he needs that too.

File: hr_grap_custom/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from odoo import fields, models
+
+
+class Employee(models.Model):
+    _inherit = "hr.employee"
+
+    # Columns Section
+    mutual_insurance_state = fields.Selection(
+        [('affiliated', 'Affiliated'),
```
Comment (robinkeunen) [inline hr_grap_custom/models/hr_employee.py]: Does this pass Black hooks? I thought it was forcing double quotes?

Comment (quentinDupont) [reply hr_grap_custom/models/hr_employee.py]: I'm a bad student and didn't learn to use black hooks yet :astonished: 
I'm gonna ask @legalsylvain to show me :)

**Comment by mclaeysb** [comment]
@quentinDupont We'd like to merge this in. Could you apply black (manually or through pre-commit) and rebase on `12.0`? Thanks!

**Comment by legalsylvain** [comment]
can be merged, no ?

**Comment by legalsylvain** [comment]
But in fact, I don't understand. quentin is proposing a grap custom module, and you'd like to use it ? if so, I think that it should be refactored in a more generic module, don't you think ? Or did I missed something ?

**Comment by robinkeunen** [comment]
@legalsylvain Yes, it can be merged, we won't use this for now. We just asked to blacken and rebase. :-)

**Comment by legalsylvain** [comment]
please, @robinkeunen, use @github-grap-bot now and do not use merge button. calling :  
``/ocabot merge option``

It will bump the revision, update the description, update the summary table of the repo, etc....
thanks !

**Comment by robinkeunen** [comment]
@legalsylvain Sorry,  not used to it yet! I'll try to think about it next time :-)

**Pull request merged**